api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -32,6 +34,16 @@ public String readBlockscoreApiKey() {
     return readToString(getCredentialsBucketName(), "blockscore-api-key.txt").trim();
   }
 
+  public String readMandrillApiKey() {
+    JSONObject mandrillKeys = new JSONObject(readToString(getCredentialsBucketName(), "mandrill-keys.json").trim());
+    return mandrillKeys.getString("api-key");
+  }
+
+  public String readMandrillFromEmail() {
```

Comment:
I've been thinking more about this and I'm less convinced that this is the right approach even though it allows us to change the address on the fly. I am now leaning toward moving this specific configuration to each of the config json files in `api/config`. It's the right thing conceptually, so let's try that.